card/mod/account/set/right/account.rb

```diff
@@ -176,8 +176,8 @@ def token_path_opts
     # Problem: when you do that then the fields are missing in the sign up form:
     # output( [field_nest(:email, view: :titled, title: "email"),
     #          field_nest(:password, view: :titled, title: "password")])
-    %({{+#{:email.cardname}|titled;title:email}}
-      {{+#{:password.cardname}|titled;title:password}})
+    %({{+#{:email.cardname}|edit_row;title:email}}
+      {{+#{:password.cardname}|edit_row;title:password}})
```

Comment:
I was hoping to see this grow more idiomatic, but it seems to be growing less.

Is there something really special about  this that justifies all the special handling?

Reply:
I think I'm understanding that this is the edit-in-place mechanism.  But I think we might want edit in place for a number of different views, no?  Eg, we might want something more like `{{+:email|labeled;edit:in-place}}`

(that `+:email` syntax might actually work now...)

Reply:
that said, I see how that might not be quick.  seems like we need a consistent way to denote wrapped content so we know what to edit in place.

Reply:
Do you have a suggestion how we call the other two values for the edit option?  (the bridge and only the content editor in a modal) Maybe it should be a little bit more self-explanatory than "bridge".

Reply:
another minor variant: `content-inline`, `content-modal`, and `full-modal`.